docs/content/dagster-plus/account/managing-users/managing-user-roles-permissions.mdx

```diff
@@ -14,32 +14,50 @@ In this guide, we'll cover how RBAC works in Dagster+, how to assign roles to us
 
 ## Dagster+ user roles
 
-Dagster+ uses a hierarchical model for RBAC, meaning that the most permissive roles include permissions from the roles beneath them. The following user roles are currently supported, in order from the **most** permissive to the **least** permissive:
-
-- Organization Admin
-- Admin
-- Editor
-- Launcher (Pro plans only)
-- Viewer
-
-For example, the **Admin** user role includes permissions specific to this role and all permissions in the **Editor**, **Launcher**, and **Viewer** user roles. Refer to the [User permissions reference](#user-permissions-reference) for the full list of user permissions in Dagster+.
+There are two key concepts for Dagster+ user roles. 
```

Comment:
I might rephrase this a little. First, I wouldn't include dates/newly introduced in docs as these docs will be around for a while. 

---

Dagster+ employs a flexible approach to user roles and permissions. This system is built on two fundamental concepts:

1. **Permission Scope**: Permissions are context-specific, falling into two main categories:
   - Organization-wide settings (for example, "Create Teams")
   - Deployment-specific actions (for example, "Launch and Cancel Backfills")

2. **Role Types**:  Dagster+ supports two types of roles:
   - **Default Roles**: Hierarchical roles, based on sensible defaults that come out of the box with Dagster.
   - **Custom Roles**: Allows you to define tailored roles with specific sets of permissions

This dual approach provides both the simplicity of predefined roles and the flexibility to create custom roles that precisely match your organization's needs.

Reply:
Is it weird that I read "based on sensible defaults that come out of the box with Dagster." in your voice? 

But serious this is super helpful, thank you, making the changes now!